test/cmd/process.sh

```diff
@@ -49,11 +49,15 @@ os::cmd::expect_success "oc process -f '${required_params}' required_param=somev
 # we should have overwritten the template param
 os::cmd::expect_success_and_text 'oc get user someval -o jsonpath={.Name}' 'someval'
 # providing a value not in the template should fail
-os::cmd::expect_failure_and_text "oc process -f '${required_params}' --value=required_param=someval --value=other_param=otherval" 'unknown parameter name "other_param"'
+os::cmd::expect_failure_and_text "oc process -f '${required_params}' --value=required_param=someval --value=other_param=otherval" 'unexpected parameter name "other_param"'
 # failure on values fails the entire call
-os::cmd::expect_failure_and_text "oc process -f '${required_params}' --value=required_param=someval --value=optional_param" 'invalid parameter assignment in'
+os::cmd::expect_failure_and_text "oc process -f '${required_params}' --value=required_param=someval --value=optional_param" 'variables must be of the form key=value or key@file'
 # failure on labels fails the entire call
 os::cmd::expect_failure_and_text "oc process -f '${required_params}' --value=required_param=someval --labels======" 'error parsing labels'
+# values can be read from files
+os::cmd::expect_success "oc process -f '${required_params}' required_param@${OS_ROOT}/test/testdata/template-value-file.txt"
+# values can be read from files using --value
+os::cmd::expect_success "oc process -f '${required_params}' --value=required_param@${OS_ROOT}/test/testdata/template-value-file.txt"
```

Comment:
both of these tests should confirm that the value from the file is reflected in the output (demonstrating that the value was actually read and used)